neurons/miners/StableMiner/main.py

```diff
@@ -17,17 +27,32 @@
 
 if __name__ == "__main__":
     try:
-        # Add the base repository to the path so the miner can access it
-        file_path: str = str(
```

Comment:
@tavurth so providing fully qualified path should probably solve this path issue but i need to test; if it doesnt i will put this path thing back.